### PR TITLE
DDF-3666 Add consistent size and alignment to 'Add Visual' menu

### PR DESCRIPTION
**Pull request by adimka**
#### What does this PR do?
Adjusts the "Add Visual" menu to provide a consistent feel.
#### Who is reviewing it? 
@mackncheesiest @Variadicism @lcrosenbu 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams/ui
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler
@bdeining
#### How should this be tested? (List steps with links to updated documentation)
Full build, manually verify menu functionality for adding visuals as well as improved aesthetics.
#### What are the relevant tickets?
[DDF-3666](https://codice.atlassian.net/browse/DDF-3666)
#### Screenshots (if appropriate)
Old Menu:
image

New Menu:
image

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.less [hunk 1]
```diff
@@ -1,31 +1,28 @@
 @{customElementNamespace}visualization-selector {
 
-    > div {
-        padding: @minimumSpacing;
-    }
+  .visualization-choice {
+    white-space: nowrap;
+    padding: @largeSpacing;
+    cursor: move; /* fallback if grab cursor is unsupported */
```
Comment (andrewkfiedler) [inline catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.less]: This cursor fallback is a perfect opportunity for a less mixin.

In addition, there are likely other preexisting areas of the code that would benefit from a cursor mixin existing.

Comment (adimka) [reply catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.less]: Done.

File: catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.less [hunk 2]
```diff
@@ -1,31 +1,25 @@
...
-        height: auto;
-        width: 100%;
-        text-align: left;
-        cursor: move; /* fallback if grab cursor is unsupported */
-        cursor: grab;
-        cursor: -moz-grab;
-        cursor: -webkit-grab;
+  .visualization-icon,
+  .visualization-text {
+    display: inline-block;
```
Comment (mackncheesiest) [inline catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.less]: ✏️ since those divs are leaf nodes anyway, I think you could leave them as `<span>`s and you wouldn't need to do this.

File: catalog/ui/catalog-ui-search/src/main/webapp/styles/mixins.less [hunk 3]
```diff
@@ -86,6 +86,14 @@
   }
 }
 
+.grab-cursor {
+  cursor: move; /* fallback if grab cursor is unsupported */
+  cursor: grab;
+  cursor: -moz-grab;
+  cursor: -webkit-grab;
+
```
Comment (mackncheesiest) [inline catalog/ui/catalog-ui-search/src/main/webapp/styles/mixins.less]: ✏️ Unnecessary whitespace

File: catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.less [hunk 4]
```diff
@@ -1,31 +1,25 @@
...
+    vertical-align: middle;
+  }
 
-        > span + span {
```
Comment (mackncheesiest) [inline catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.less]: Yeesh, that original selector though.

File: catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.hbs [hunk 5]
```diff
@@ -11,23 +11,24 @@
...
+    <div class="visualization-text">3D Map</div>
+</div>
+<div class="visualization-choice choice-inspector is-button" data-choice="inspector">
+    <div class="visualization-icon fa fa-info"></div>
+    <div class="visualization-text">Inspector</div>
+</div>
+<div class="visualization-choice choice-histogram is-button" data-choice="histogram">
+    <div class="visualization-icon fa fa-bar-chart"></div>
+    <div class="visualization-text">Histogram</div>
+
```
Comment (mackncheesiest) [inline catalog/ui/catalog-ui-search/src/main/webapp/component/visualization-selector/visualization-selector.hbs]: ✏️ Unnecessary whitespace

**Comment by adimka** [comment]
build now

**Comment by adimka** [comment]
build now

**Comment by adimka** [comment]
build now

**Comment by adimka** [comment]
build now
